test/e2e/nightfall-sdk.test.mjs

```diff
@@ -733,6 +1295,73 @@ describe('Testing the Nightfall SDK', () => {
     });
   });
 
+  describe('Restoring tokens', () => {
```

Comment:
`Restoring tokens`, please explain the purpose of this test

Reply:
It’s transferring the tokens back to the original account. For NFT it’s mandatory, otherwise we test would fail the second time it’s launched as user doesn’t have that specific token Id. For the rest of tokens, it’s good practice because in testnet launching this test many times would make a user run out of tokens.

Reply:
So this is done to run the same tests again with restarting nightfall ? Essentially making the test file idempotent?

Also, can we add some comments on top for situations such as this please, would help while reviewing or for whoever works in the same test file later?

Reply:
done